src/translib/acl_app.go

```diff
@@ -313,6 +315,83 @@ func (app *AclApp) translateCRUCommon(d *db.DB, opcode int) ([]db.WatchKeys, err
 	return keys, err
 }
 
+func (app *AclApp) processAclGet(dbs [db.MaxDB]*db.DB) error {
+
+	var err error
+	var topmostPath bool = false
+
+	d := dbs[db.ConfigDB]
+	acl := app.getAppRootObject()
+
+	log.Infof("processAclGet--Path Received: %s", app.pathInfo.Template)
+	targetType := reflect.TypeOf(*app.ygotTarget)
```

Comment:
If we are doing all this to set topmostPath, could we not simply use the targetUriPath or the app.pathInfo.Template?

Reply:
Probably yes but this  is how they are determining the top level path in other places. It  basically does a  sanity check and also prints a log message.